src/main.cpp

```diff
@@ -69,6 +69,22 @@ std::string exec(const char* cmd) {
     return result;
 }
 
+void open_output_file(const std::string& a, const std::string& b)
+{
+    std::filesystem::path path(b);
+    if (!a.empty()) {
+        path = a + '.' + b;
+    }
+    try {
+        globals::instance()._fp.open(path);
+    }
+    catch (const std::exception& e) {
+        cout_safe([&](auto& s){
```

Comment:
Is this a fatal scenario? What happens in this case to the output? I suppose if it drops on the floor the tool can still run, but it won't be worth much?

Reply:
Agreed; that code was pulled in, but it should fail.